ls1intum/Artemis
# `General`: Fix wrong display of participation status

### Checklist
#### General


#### Client

### Motivation and Context


Fix some errors introduced in the latest release.

### Description

Fixes three bugs:
- When starting an exercise the submission status (which would normally show the score), now correctly shows "no submission"
- When resuming an old exercise, the resume button is correctly replaced by the clone button, once the resuming is done
- When participating in a file upload exercise, the participation status always showed "not submitted"

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Student
- 1 normal Programming Exercise
- 1 old Programming Exercise with a participation
- 1 Fileupload exercise

1. Log in to Artemis
2. Participate in the normal exercise with a user that did not participate before
3. Check that "no submission" is correctly displayed

4. Navigate to the old exercise
5. Resume the old exercise
6. Check that the buttons are correctly displayed

7. Navigate to the Fileupload exercise
8. Participate and submit a file
9. Check that Artemis shows that a file was submitted

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
